packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-comment-box.tsx

```diff
@@ -475,6 +504,12 @@ export const PullRequestCommentBox = ({
                   }
                 }}
               />
+
+              {showAiLoader && (
```

Comment:
NIT: Instead of blocking the textarea, should we display a loader within the button to indicate loading state?

Thoughts?